code/__DEFINES/maps.dm

```diff
@@ -64,11 +75,24 @@ require only minor tweaks.
 #define ZTRAITS_CENTCOM list(ZTRAIT_CENTCOM = TRUE)
 #define ZTRAITS_STATION list(ZTRAIT_LINKAGE = CROSSLINKED, ZTRAIT_STATION = TRUE)
 #define ZTRAITS_SPACE list(ZTRAIT_LINKAGE = CROSSLINKED, ZTRAIT_SPACE_RUINS = TRUE)
-#define ZTRAITS_LAVALAND list(\
+#define ZTRAITS_MININGBASIC list(\
     ZTRAIT_MINING = TRUE, \
     ZTRAIT_LAVA_RUINS = TRUE, \
-    ZTRAIT_BOMBCAP_MULTIPLIER = 2, \
-    ZTRAIT_BASETURF = /turf/open/lava/smooth/lava_land_surface)
+    ZTRAIT_BOMBCAP_MULTIPLIER = 2)
+
+#define ZTRAITS_LAVAWORLD list(\
+	ZTRAIT_BASETURF = /turf/open/lava/smooth/lava_land_surface, \
+	ZTRAIT_THEMATIC_PLATING = /turf/open/floor/plating/asteroid/basalt/lava_land_surface, \
+	ZTRAIT_THEMATIC_LAVA = /turf/open/lava/smooth/lava_land_surface, \
+	ZTRAIT_WEATHER_ASHSTORM = TRUE )
+	//ZTRAIT_MINERAL_ICON = not included because lavaworld uses the default
+
+#define ZTRAITS_SNOWWORLD list(\
+	ZTRAIT_BASETURF = /turf/open/lava/plasma/lava_land_surface, \
+	ZTRAIT_THEMATIC_PLATING = /turf/open/floor/plating/asteroid/snow/lava_land_surface, \
```

Comment:
this entire path makes me sad. why is lava_land_surface a child of two types.

Reply:
turf pathing is fucked, this is nothing new

Reply:
dont make it worse :^(